### PR TITLE
Add Development Steps in README

### PR DESCRIPTION
**Pull request by WithSJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,25 @@ csq <filename>
 -   Don't commit two different things together.
 -   After all things push your code to our **main branch**.
 
+
+## Development Steps
+
+### 1. Data Types
```
Comment (AniketKumar2500) [inline README.md:37]: Already datatypes are there like STR, FLOAT and LIST
And the boolean is expressed in terms of FLOAT only so please see it.

**Review by AniketKumar2500 — COMMENTED** [review]
Already datatypes are there like STR, FLOAT and LIST
And the boolean is expressed in terms of FLOAT only so please see it.

**Comment by WithSJ** [comment]
> Already datatypes are there like STR, FLOAT and LIST And the boolean is expressed in terms of FLOAT only so please see it.



"We need integers because floating-point values consume more memory and take more time to process. For readability, we should use booleans instead of 0 and 1, as it can be difficult to understand. Currently, it seems that the string data type is not fully developed. Therefore, we can define all data types using classes like this:

```

int_value := int() 
bool_value := bool() 
str_value := str() 
```
int  - (Default value: 0)
bool - (Default value: false)
str - (Default value: '')

In the CSQ language, we can achieve this by using classes, and within these classes, we can define methods like str().remove(), str().replace(), and int().to_str()."
in the end in class we need basic type of data types

**Comment by AniketKumar2500** [comment]
So we must do it after the development of classes.

**Comment by WithSJ** [comment]
No. Data Type should be devlope  in cpp and other operations and methods like i mention for that we make class

**Comment by AniketKumar2500** [comment]
> No. Data Type should be devlope in cpp and other operations and methods like i mention for that we make class

But you currently you could only use functions written in Cpp but not classes if done so will not be able to access it's members.

**Pull request closed without merging**